starter-core/src/main/java/io/micronaut/starter/feature/other/template/readme.rocker.raw

```diff
@@ -6,6 +6,7 @@ Feature feature
 
 @if (feature.getMicronautDocumentation() != null || feature.getThirdPartyDocumentation() != null) {
 
+
```

Comment:
without this, I was seeing errors in `DiffControllerSpec` where it seemed to be putting the titles at the end of the line above... 🤔